packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -246,14 +253,6 @@ abstract class BindingBase {
         },
       );
 
-      registerStringServiceExtension(
-        name: 'connectedVmServiceUri',
-        getter: () async => connectedVmServiceUri ?? '',
-        setter: (String uri) async {
-          connectedVmServiceUri = uri;
-        },
-      );
-
       registerStringServiceExtension(
         name: 'activeDevToolsServerAddress',
```

Comment:
this one probably needs to move out of the assert block too right?

Reply:
Ahh good catch